### ISSUE TITLE
Add `BaseSell` values to all items that can be sold and do not have a base sell price.

### ISSUE THREAD

**Issue by mrhappyasthma** [project-topaz/topaz#409]
Here's a list of all of the items that need to be updated:

[item_basic.xlsx](https://github.com/project-topaz/topaz/files/4316542/item_basic.xlsx)

```
select * from item_basic where BaseSell = 0 and NoSale = 0
```

Related issues: https://github.com/project-topaz/topaz/issues/187

**Comment by ibm2431** [comment]
This is definitely going into my drawer for next free period, thanks!

Tagging: @project-topaz/researcher and @Wiggo32

**Comment by mrhappyasthma** [comment]
Warning: there's more than 5500 items returned by this query. So it may take some time to update them all.

**Comment by mrhappyasthma** [comment]
FFXIclopedia shows `Resale Price` under its items, however they may not be 100% accurate.

https://ffxiclopedia.fandom.com/

**Comment by UynGH** [comment]
That's definitely one issue I can look into for the next free campaign. I'll check my pricelog (thanks to ibm2431) data to compare for now. Thanks, mrhappyasthma.

**Comment by 59blargedy** [comment]
Just to check before I start trying to collect - I assume we need this data with 0 fame? and what goes in the nosale column?

**Comment by mrhappyasthma** [comment]
It seems like `NoSale` is set to 1 for `EX` items that cannot be sold.

So I believe we only need prices for items that are sellable (i.e. `NoSale = 0`).

I imagine `BasePrice` should be the 0 fame value. Although looking through the code, it does not look like we take fame into account at all (unless I just couldn't find where that happens). But it's something we could more easily apply to the base price by scaling it up by some factor.

**Comment by TeoTwawki** [comment]
> I imagine BasePrice should be the 0 fame value. 

yes. in retail fame adds a small bit of extra gil to this value in areas where fame applies.

**Comment by 59blargedy** [comment]
i started updating here - feel free to update:
https://docs.google.com/spreadsheets/d/1fPJAoyxn1JuHXWTLS9ccF_5Coe9cQTniOJRbZwg_9Zc/edit?usp=sharing
added a column for acquisition method, as many of these are inaccessible without gp or on certain times of the year 
for things that are rare/ex, im changing no sale to 1
